experimental/ixdtf/src/parser.rs

```diff
@@ -0,0 +1,400 @@
+#[derive(Clone, Copy)]
+enum  DateTimeSeparator {
+    CapitalT,
+    LowCaseT,
+    Space,
+}
+
+impl DateTimeSeparator {
+    fn value(&self) -> String {
```

Comment:
Suggestion: return `&'static str` instead of `String` to avoid the extra allocation.

In general, the parser should require no allocations. You can achieve this by making your crate `#![no_std]` and not enabling the "alloc" feature (or making it optional).